### PR TITLE
show lesson/status

### PR DESCRIPTION
**Pull request by minh93**

### CODE REVIEW

File: app/controllers/lessons_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    end
+  end
+
+  def show
+    @lesson = Lesson.find params[:id]
+  end
+
+  def index
+    @lessons = current_user.lessons.
+      paginate page: params[:page], per_page: 10
```
Comment (yuyuvn) [inline app/controllers/lessons_controller.rb]: dấu chấm bên dưới

File: config/locales/en.yml [hunk 2]
```diff
@@ -85,3 +89,12 @@ en:
         confirm: "Are you sure?"
       index:
         h1index: "All users"
+  lessons:
+    lesson:      
+      take_lesson: "Take"
+      question_done: "/Total"
+      slash: "/"
```
Comment (yuyuvn) [inline config/locales/en.yml]: don't need i18n this

File: app/models/result.rb [hunk 3]
```diff
@@ -2,4 +2,8 @@ class Result < ActiveRecord::Base
   belongs_to :lesson
   belongs_to :word
   belongs_to :answer
+
+  scope :answered, ->{where.not answer_id: nil}
+  scope :unanswered, ->{where answer_id: nil}
+  scope :correct_answer, ->{joins(:answer).where :answers => {is_correct: true}}
```
Comment (dieunb) [inline app/models/result.rb]: convention

File: app/views/lessons/_lesson.html.erb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  <div class="col-md-6">    
+    <div class="col-md-2">
+      <%= lesson.results.answered.count %>
+      <%= t ".question_done" %>
+    </div>
+    <div class="col-md-4">
+      <%= lesson.created_at.to_formatted_s :short %>
+    </div>
+  </div>
+  <% if !lesson.finished? %>
```
Comment (dieunb) [inline app/views/lessons/_lesson.html.erb]: unless

File: app/controllers/lessons_controller.rb [hunk 5]
```diff
@@ -10,7 +12,19 @@ def create
     end
   end
 
-  def show
+  def show    
+  end
+
+  def index
+    if logged_in?
```
Comment (dieunb) [inline app/controllers/lessons_controller.rb]: Verify user and after you don't care it

File: app/controllers/lessons_controller.rb [hunk 6]
```diff
@@ -10,7 +14,21 @@ def create
...
-  def show
+  def show    
+  end
+
+  def index
+    @lessons = current_user.lessons
+      .paginate page: params[:page], per_page: 10
+  end
+
+  def current_lesson
```
Comment (dieunb) [inline app/controllers/lessons_controller.rb]: private method

File: app/controllers/lessons_controller.rb [hunk 7]
```diff
@@ -10,7 +14,21 @@ def create
...
     @lesson = Lesson.find params[:id]
   end
+
+  def correct_user
```
Comment (dieunb) [inline app/controllers/lessons_controller.rb]: private method

File: app/controllers/lessons_controller.rb [hunk 8]
```diff
@@ -10,7 +14,21 @@ def create
     end
   end
 
-  def show
+  def show    
+  end
+
+  def index
```
Comment (dieunb) [inline app/controllers/lessons_controller.rb]: Chú ý thứ tự hàm

**Pull request merged**